### DISCUSSION TITLE
Changing editor playback speed also affects test play speed

### DISCUSSION THREAD

**Discussion by 12beesinatrenchcoat** [ppy/osu#20671] [Q&A] [answered]
No clue if this is intentional or not, but when changing the playback speed in the editor, the test play speed is also affected. This doesn't happen in stable, so I thought I should bring it up.

https://user-images.githubusercontent.com/25379179/194777387-c85d1e4c-25e1-4497-8efc-16737a3d0ae9.mp4

Logs, just in case:
[performance-update.log](https://github.com/ppy/osu/files/9742469/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9742470/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9742471/updater.log)
[database.log](https://github.com/ppy/osu/files/9742472/database.log)
[network.log](https://github.com/ppy/osu/files/9742473/network.log)
[performance.log](https://github.com/ppy/osu/files/9742474/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/9742475/performance-draw.log)

**Comment by peppy** [comment, marked as answer]
This is a bit of a side effect from how we are managing beatmap tracks and screens... and everything in general. The editor has no easy way to "unapply" its changes before showing the test player.

I think it's easiest to leave this the way it is until there are more complaints, as changing is non-trivial.
